src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,9 +19,13 @@ public User login(String login, String password) throws AuthenticationException
     }
 
     private User findByLogin(String login) {
+        log.debug("Service level. Find by login was called with params: {}", login);
         User user = new User(login, "1234");
         // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        log.debug("Service level. Found user by id: {}, user's data {}",
+                user.getUserId(), user);
         return user;
     }
 }
+
```

Comment:
Don't forget about empty lines in the end of files. There should be exactly one empty line at the end of the file.